Resources/Private/Templates/News/GoogleNews.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-stylesheet type="text/xsl" href="{xslFile}"?>
+
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+		xmlns:news="http://www.google.com/schemas/sitemap-news/0.9"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
+        http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd
+        http://www.google.com/schemas/sitemap-news/0.9
+        http://www.google.com/schemas/sitemap-news/0.9/sitemap-news.xsd">
+	<f:for each="{items}" as="item">
+		<f:if condition="{item.loc}">
+			<url>
+				<loc>{item.loc}</loc>
+				<news:news>
+					<news:publication>
+						<news:name>Austrian Wings - Österreichs Luftfahrtmagazin</news:name>
```

Comment:
can we reuse the setting from list.xml, same also for the language

Reply:
of course... I just realized we still have AW in there :-)